clickfile.py

```diff
@@ -1265,18 +1265,45 @@ def build(tag):
         sys.exit(command_build.returncode)
 
 
-@k6.command("run", help="Run k6 performance test")
+@k6.command("prepare-accounts", help="Generates users and fund its account balances.")
 @click.option("-n", "--network",required=True, default="local", help="Which network to use for envs assignment")
-@click.option("-s", "--script", required=True, default="./loadtesting/k6/tests/sendNeon.test.js", help="Path to k6 script")
-@click.option("-u", "--users", default=None, required=True, help="Number of users (will be generated before load test run)")
+@click.option("-u", "--users", default=None, required=True, help="Number of users to generate before test run")
 @click.option("-b", "--balance", default=None, required=True, help="Initial balance of each user in Neon")
-@click.option("-a", "--bank_account", default=None, required=False, help="Bank account address")
+@click.option("-a", "--bank_account", default=None, required=False, help="Bank account address if needed")
+@catch_traceback
+def prepare_accounts(network, users, balance, bank_account):
+    network_object = network_manager.get_network_object(network)
+    web3_client = NeonChainWeb3Client(proxy_url=network_object["proxy_url"])
+    faucet = Faucet(faucet_url=network_object['faucet_url'], web3_client=web3_client)
+    account_manager = EthAccounts(web3_client, faucet, bank_account)
+    k6_prepare_accounts(account_manager, users, balance)
+
+
+@k6.command("deploy-erc20", help="Deploy erc20 contract.")
+@click.option("-n", "--network",required=True, default="local", help="Which network to use for envs assignment")
+@click.option("-b", "--balance", default=None, required=True, help="Initial balance of erc20 owner")
+@click.option("-a", "--bank_account", default=None, required=False, help="Bank account address if needed")
 @catch_traceback
-def run(network, script, users, balance, bank_account):
+def deploy_erc20(network, balance, bank_account):
```

Comment:
are you sure that we need all this commands to prepare data? 
I suggest to have one command for "run load tests" which will include all needed preparation  (contracts deploy, account creating e.t.c)

Reply:
done